polars/polars-lazy/Cargo.toml

```
@@ -24,7 +24,7 @@ polars-pipe = { version = "0.26.1", path = "./polars-pipe", optional = true }
 polars-plan = { version = "0.26.1", path = "./polars-plan" }
 polars-time = { version = "0.26.1", path = "../polars-time", optional = true }
 polars-utils = { version = "0.26.1", path = "../polars-utils" }
-pyo3 = { version = "0.16", optional = true }
+pyo3 = { version = "0.18", optional = true }
```

Must this include the bug-fix number? I.e. must it be:
```suggestion
pyo3 = { version = "0.18.0", optional = true }
```
?

---

I prefer 0.18 as this makes it more clear that we accept future patches.